images/win/scripts/Installers/Install-RootCA.ps1

```diff
@@ -104,6 +104,27 @@ function Disable-RootAutoUpdate {
     Set-ItemProperty $regPath -Name $regKey -Type DWord -Value 1
 }
 
+function Invoke-WithRetry {
```

Comment:
move this function before Import-SSTFromWU

Reply:
done